src/rpc/blockchain.cpp

```diff
@@ -2334,30 +2362,117 @@ UniValue CreateUTXOSnapshot(
         tip = CHECK_NONFATAL(chainstate.m_blockman.LookupBlockIndex(stats.hashBlock));
     }
 
-    LOG_TIME_SECONDS(strprintf("writing UTXO snapshot at height %s (%s) to file %s (via %s)",
-        tip->nHeight, tip->GetBlockHash().ToString(),
-        fs::PathToString(path), fs::PathToString(temppath)));
-
-    SnapshotMetadata metadata{tip->GetBlockHash(), stats.coins_count, tip->nChainTx};
+    switch (format) {
+    case UTXOSnapshotFormat::COMPACT:
```

Comment:
The code blocks below each case are indented 4 spaces too deeply (don't hesitate to run clang-format on the diff).
```suggestion
    case UTXOSnapshotFormat::COMPACT: {
```

Reply:
Thanks. I had removed a formatting change to make review a bit easier and then was not running it on subsequent changes.